### ISSUE TITLE
Control Button Names are Read out twice by the Screen Reader

### ISSUE THREAD

**Issue by sarbbottam** [videojs/video.js#7450]
## Description
Control Button Names are Read out twice by the Screen Reader.
Tested on Edge, Chrome on Mac with VoiceOver

_I could not reproduce the issue on Safari/Mac with VoiceOver_

Please `unmute` the below video to listen to the VoiceOver feedback.
https://user-images.githubusercontent.com/949380/135203499-62dddc23-16e9-425c-844e-7fc0d300d8c4.mp4

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Start the local server via `npm start`
Refer to [Running a local web server ](https://github.com/videojs/video.js/blob/main/CONTRIBUTING.md#testing-locally) section of the [CONTRIBUTING.md](https://github.com/videojs/video.js/blob/main/CONTRIBUTING.md#testing-locally) for further details.
2. Once the server is up and running access the local instance at http://localhost:9999/sandbox/index.html
3. Start VoiceOver and operate the Video Player that is rendered at http://localhost:9999/sandbox/index.html
4. Navigate to the Play/Pause button by pressing `tab`
5. Activate the the button by pressing `enter`

### Expected
The control names are read out once

### Actual
The control names are read out twice

### Error output
NA

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
>what version of videojs does this occur with?

Latest

#### browsers
Chrome/ Edge
#### OSes
Mac
I believe this issue is observed on Windows too.

### plugins
>are any videojs plugins being used on the page? If so, please list them below.

NA

**Comment by OwenEdwards** [comment]
I agree that this is an issue, but we need a broader review and a plan to test all combinations of OS, browser, and SR after the change. #7451 might be a good start, but there may be some components that need to add aria-live back to the control if it is removed in the `ClickableComponent` - that's the broader review that needs to happen.

**Comment by misteroneill** [comment]
@Essk This sounds like the issue you were investigating internally at Brightcove.

**Comment by Essk** [comment]
Yes indeed, this is closely adjacent to what I was looking at.
My investigation was around SR output immediately after interacting with the Big Play button.  The main issue highlighted was that multiple controls are announced when the BPB is interacted but it was also noted that some controls are read out twice and it was suggested that we might remove `aria-live` to prevent this.

I went ahead and tested what happened across a few SR/browser combos with `aria-live` disabled and always force-focussing the  play-pause toggle (the second test is not related to controls being announced twice but I did it at the same time for the BC tickets): 
https://docs.google.com/spreadsheets/d/1bwR5KQkCZzrdvU5Nzx9QYrHJmxfEWq3_8NaixZpBanQ/edit#gid=0

Right now, I see two problems that would need to be worked-around / overcome if `aria-live` were removed from some / all controls:
1) From my test runs, it looks like `aria-live` is _necessary_ for controls to be announced _at all_ after BPB interaction in Chrome & FF with NVDA & JAWS. This may change if we also change focus behaviour or apply `role=application` to the top-level container (which should cause NVDA and JAWS to properly relay a space / enter press instead of a click).
2) I am mindful of this [WCAG guidance](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html) that states that content that appears on hover (ie the controls) should be perceivable to assistive technology. There would need to be more testing to be 100% sure on this but I think applying `aria-live` to the controls might be the _only_ way to be certain of this perceivability in the majority of SR/browser combos.
